config/images.yml

```diff
@@ -360,9 +340,6 @@
 "static/content/blog/thumbnails/stories-generic.jpg":
   alt: Students enjoying a lesson in a classroom
 
-"static/content/blog/thumbnails/stories-katie.jpg":
```

Comment:
Need to keep this if we keep the international returners blog temporarily @philspray ?